RileyLinkBLEKit/RileyLinkBLEDevice.m

```diff
@@ -278,7 +278,7 @@ - (void)checkVersion {
         NSString *foundVersion;
         BOOL versionOK = NO;
         // We run two commands here, to flush out responses to any old commands
-        [s doCmd:cmd withTimeoutMs:1000];
+        [s doCmd:cmd withTimeoutMs:5000];
```

Comment:
I was seeing occasional timeouts on that command.  I've been trying to eliminate all the RileyLinkTimeouts, as they shouldn't really occur unless something is wrong with the connection to the RileyLink.   As this only happens on startup, and normally returns fast, it shouldn't have any negative side effects, and it may give the OS more time to set up all the bluetooth stuff.  If we do get a timeout here, it not flushing as intended, and the get version was failing.